src/docs/Welcome.stories.mdx

```diff
@@ -15,13 +15,13 @@ import { Meta } from "@storybook/addon-docs";
 
 Welcome to the conda-store-ui project documentation! conda-store-ui is a frontend for conda-store, allowing interaction with environments through its interface. 
 
-As a pre-requisite, you will need to have access to a running instance of conda-store.
+As a pre-requisite, you will need to have access to a running instance of conda-store-server.
```

Comment:
Is it a pre-requisite if the conda-store-ui package will launch a conda-store-server for you? This statement makes me think that I need to go figure out conda-store server before coming to the ui. Which I dont *think* is true?

Reply:
no, it isn't. Good catch.